### PR TITLE
Applied my own rendition for the Home Page (please feel free to modify)

### LINKED ISSUE #25: [REFACTOR] Apply Home Page and HeroNavbar rendition to project
- Applied my own rendition for the Home Page (please feel free to modify)
- Parallax effect 
- Dynamic color for HeroNavbar logo
- Create an initial content section for Home page

### PR DESCRIPTION
**Pull request by MAKIENAUT**
Applied my own rendition for the Home Page (please feel free to modify)
Parallax effect
Dynamic color for HeroNavbar logo
Create an initial content section for Home page

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,53 +1,18 @@
...
+import { useInView } from "@/hooks/useInView";
 
 export default function Home() {
-  const [ref, inView, entry] = useInView({ threshold: 0.95 });
-
-  // const res = await getPosts();
-  // console.log("Response from getPosts:", res);
+  const contentRef = useRef<HTMLDivElement>(null);
+  const thresholds = Array.from({ length: 100 }, (_, i) => i / 100);
+  const { intersectionRatio } = useInView(contentRef, thresholds);
```
Comment (Noxaelum) [inline src/app/page.tsx:10]: What does this do?

Comment (MAKIENAUT) [reply src/app/page.tsx:10]: reference for the custom inview hook to be able to detect the visibility of an element

Comment (Noxaelum) [reply src/app/page.tsx:10]: Ah, so gumawa ka ng custom hook using the native intersection observer API

Comment (MAKIENAUT) [reply src/app/page.tsx:10]: not my idea, but yeah.. works like a charm

**Comment by Noxaelum** [comment]
Looks good to me. Nothing seems out of the ordinary

**Pull request merged**